tests/v2/integration/steveapi/json/user-a_none_fieldSelector=metadata.name=test1.json

```diff
@@ -1,8 +1,7 @@
 {
   "actions": {},
-  "count": 5,
```

Comment:
It looks from changes like this that this may have been cherry-picked directly from the 2.6 branch? The responses in 2.7 will be slightly different because of the new 2.7-only features, so it will need to be re-run against a 2.7 cluster before committing.

Reply:
Oops I'll update it and re run

Reply:
Updated.